src/assets/styles/components/sidebar-workspace.scss

```diff
@@ -0,0 +1,141 @@
+.sidebar-workspace {
+  position: sticky;
+  margin: 40px 0 0;
+  max-height: 90vh;
+  padding: 0 16px;
+  width: 256px;
+  height: 100vh;
+
+  .top-sidebar-container {
+     font-size: 14px;
+     line-height: 20px;
+
+     .top-sidebar {
+      border-bottom: 1px solid var(--ds-border);
+      list-style: none;
+      margin: 0;
+      margin-bottom: 5px;
+      padding: 0;
+
+      .top-list-item {
+        display: flex;
+        align-items: center;
+        border-radius: 8px;
+        min-height: 40px;
+        padding: 6px 8px;
+        margin: 0 0 0 4px;
```

Comment:
margin-bottom: 4px. it looks like you copy from trello, please do not copy from trello, try to write yourself